client/src/modules/grades/templates/action.tmpl.html

```diff
@@ -0,0 +1,22 @@
+<div class="ui-grid-cell-contents text-center" uib-dropdown dropdown-append-to-body>
+  <a uib-dropdown-toggle href>
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-action="{{rowRenderIndex}}"  class="dropdown-menu-right" uib-dropdown-menu>
```

Comment:
You should use `bh-dropdown-menu-auto-dropup` here to make sure that if there are many grades, the dropdown will continue to be visible.